### ISSUE TITLE
Where is mpc_trainer.py used in tf2rl?

### ISSUE THREAD

**Issue by cubicgate** [keiohta/tf2rl#89]

**Comment by keiohta** [comment]
Hi @cubicgate, it is used in `examples/run_mpc.py`. You can use as:
```
$ python examples/run_mpc.py
```

**Comment by cubicgate** [comment]
Thanks @keiohta ! I see it now. A reward function is usually defined as r(s,a), but yours in run_mpc.py is defined as r(s,s', a).  The paper Neural Network Dynamics for Model-Based Deep Reinforcement Learning with Model-Free Fine-Tuning https://arxiv.org/pdf/1708.02596.pdf  also used r(s,a). How did you create the math formula for r(s,s',a)?   Thanks!

**Comment by keiohta** [comment]
This seems to be a misimplementation! I'll check theory and fix it if I need. Thanks!

**Comment by cubicgate** [comment]
Hi @keiohta Your reward_fn_pendulum(obses, next_obses, acts) does not actually use next_obses. You used the exact math formula for Penulum-v0 from gym.

**Comment by keiohta** [comment]
Yeah, I think you're right. I reflect your kind suggestion to the latest master as above commit.
Thanks @cubicgate !
